docs/software/system_stats.md

```diff
@@ -93,6 +94,27 @@ DietPi-Dashboard is a very lightweight and standalone web interface for monitori
     journalctl -u dietpi-dashboard
     ```
 
+=== "Password Protection"
+
+    Password protection is enabled by default from DietPi 7.9 and on, but if you installed it before, you'll need to make these changes in the config file:
+    ```toml
+    pass = true
+    # You can get the hash by running "echo -n '<PASSWORD>' | sha512sum
+    hash = "SHA512 hash of password"
+    # You can get a random secret by running "openssl rand -hex 32"
+    secret = "Random 64-character secret"
+    ```
+
+    Then, restart the service.
+
+=== "Resetting the password"
+
+    To reset the password, you will need to change the `hash` setting in the config file. See the password protection section for how to get the correct hash.
+
+=== "Invalidating logged in sessions"
+
+    If you need to make every logged in session log out, but you don't want to change the password, you can instead change the secret. Generate a new secret (see password protection), and put it in the config file. Then, make sure to restart the service. This will log everyone out, and then they will need to log back in to continue.
+
```

Comment:
I think these three tabs should be one. it's a bit overkill otherwise and we need to refer from one tab to the other. It's all about password protection and related sessions.

Btw, important IMO to have the hash and random secret commands in the config file as well. I totally forgot about that 👍.